cargo-registry-index/lib.rs

```diff
@@ -125,6 +125,8 @@ pub struct Crate {
     pub yanked: Option<bool>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub links: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub rust_version: Option<String>,
```

Comment:
@Turbo87 I wanted to double check your thoughts on the fact that the index field (`rust_version`) doesn't correspond with the manifest field (`rust-version`).

I had missed that the struct doesn't have  `#[serde(rename_all = "kebab-case")]` set on it and didn't notice this in any of the tests.  Apparently this is our first field with a word separator.

See
```json
{"name":"epage-publish-test-lib","vers":"0.1.0","deps":[],"cksum":"4ed5dc06fc83278e1b50cd58ceab528f4e1b24fbf675cc326af2d429bb7b586e","features":{},"yanked":false,"rust_version":"1.64.0"}
```
(I was verifying we didn't accidentally break the entire ecosystem....)

Reply:
Ah, missed that field